docs/source/notebooks/model_comparison.ipynb

```diff
@@ -1,25 +1,34 @@
 {
```

Comment:
<p>The first of <s>this</s> <strong>these</strong> functions is <code>compare</code>, <strong>which</strong> <s>this one</s> computes <strong>LOO (or WAIC)</strong> <s>WAIC (or LOO)</s> from a set of traces and models and returns a DataFrame.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>